homeassistant/components/trafikverket_ferry/coordinator.py

```diff
@@ -65,11 +65,13 @@ async def _async_update_data(self) -> dict[str, Any]:
         """Fetch data from Trafikverket."""
 
         departure_day = next_departuredate(self._weekdays)
-        currenttime = datetime.now()
+        currenttime = datetime.now(get_time_zone(self.hass.config.time_zone))
```

Comment:
Please use `now` in our dt util instead.

Reply:
> Please use `now` in our dt util instead.

Is there a difference since it doesn't appear to be one from the normal datetime.now?